lib/bom.js

```diff
@@ -1324,3 +1324,17 @@ declare class VRDisplayEvent extends Event {
 
   constructor(type: string, eventInitDict: VRDisplayEventInit): VRDisplayEvent;
 }
+
+declare type MediaQueryListEvent = {
+  matches: boolean,
+  media: string,
+};
+
+declare type MediaQueryListListener = MediaQueryListEvent => void;
```

Comment:
> A MediaQueryListListener is a callback function that gets invoked with a single argument, the MediaQueryList for which the evaluated result changed.

From https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryListListener

And

> [MediaQueryListEvent] instances are available as the event object on a function referenced by a MediaQueryList.onchange property or MediaQueryList.addEvent() call.

From https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryListEvent

So I think the callback passed to `addListener` and `removeListener` should just take a `MediaQueryList` -- `onchange` or `addEvent` are the only things that involve the `MediaQueryListEvent` type. Does that seem right?

Reply:
But the [addListener](https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener) docs specifically say the callback receives a `MediaQueryListEvent` (emphasis mine):

> A function or function reference representing the callback function you want to run when the media query status changes. In the original implementation, the callback was a non-standard MediaQueryListListener object. In the new implementation the standard event mechanism is used, the callback is a standard function, and **the event object is a MediaQueryListEvent**, which inherits from Event.

I think the docs for `MediaQueryListEvent` itself are just a bit unclear/incomplete.